app/controllers/v0/disability_ratings_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+module V0
+  class DisabilityRatingsController < ApplicationController
+    skip_before_action :authenticate
+
+    def show
+      disability_rating = DisabilityRating.new(current_user.rating_record['disabilityRatingRecord'])
```

Comment:
The disability rating object/serializer shouldn't need to know how to access the disability rating information from the rating record, they just get passed the data that is applicable. We could just have an endpoint for returning the whole rating record but I didn't see any use cases for the other data at this time